### PR TITLE
Initial Support for Airspeed Velocity

### PR DESCRIPTION
**Pull request by richardliaw**
This is to address a variety of regression testing TODOs, and also should partially address #2112.

### CODE REVIEW

**Review by pcmoritz — COMMENTED** [review]
Can you add a simple Benchmarks.md file that tells people how to run the benchmarks?

**Comment by pcmoritz** [comment]
Thanks, there was another linting error which I fixed now, I'll merge it when the tests pass (https://travis-ci.com/robertnishihara/ray-private-travis/builds/74152646)

File: python/benchmarks/benchmarks.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from __future__ import print_function
```
Comment (robertnishihara) [inline python/benchmarks/benchmarks.py]: This should be 

```
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
```

File: python/README-benchmarks.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Running the benchmarks
```
Comment (robertnishihara) [inline python/README-benchmarks.md]: In general, we should use rst, not markdown

File: python/benchmarks/benchmarks.py (hunk 1 above)
Comment (robertnishihara) [reply python/benchmarks/benchmarks.py]: I realize this is autogenerated code.

**Comment by pcmoritz** [comment]
Jenkins retest this please

**Review by pcmoritz — APPROVED** [review]

**Pull request merged**